src/main/java/org/fcrepo/camel/FcrepoProducer.java

```diff
@@ -216,9 +219,60 @@ protected String getUrl(final Exchange exchange) {
         } else if (method == HttpMethods.DELETE && endpoint.getTombstone()) {
             url.append("/fcr:tombstone");
         }
+
         return url.toString();
     }
 
+    /**
+     *  Given an exchange, extract the Prefer headers, if any.
+     *
+     *  @param exchange the incoming message exchange
+     */
+    protected String getPrefer(final Exchange exchange) {
```

Comment:
It does not appear that this method is being used outside of this class. If that is indeed the case, please make it `private`.